.tekton/pull-request.yaml

```diff
@@ -37,26 +37,48 @@ spec:
           - name: depth
             value: "0"
       - name: build-appstudio-utils
-        runAfter:
-          - fetch-repository
+        matrix:
+          params:
+          - name: PLATFORM
+            value:
+            - linux/x86_64
+            - linux/arm64
         params:
           - name: IMAGE
-            value: quay.io/konflux-ci/pull-request-builds:appstudio-utils-{{revision}}
+            value: quay.io/konflux-ci/appstudio-utils:{{ revision }}
```

Comment:
I just now noticed - what is the purpose of this change? I don't think we want to push to the "production" repo from the PR pipeline

Reply:
Ah, I think it's fixed in a later commit